src/accessibility/index.js

```diff
@@ -0,0 +1,30 @@
+export type InstructionsContext = { isSearchable?: boolean, isMulti?: boolean, label?: string };
+export type ValueEventContext = { value: string };
```

Comment:
I think it's actually better here to pass in value as an object and the function getOptionLabel as a function argument here. 

This solves for Creatable select, where we need to be able to either fire an announcementEvent from the wrapping Creatable HoC, or infer whether or not a value isNew to change the returned event string. However this also requires that the user apply getOptionLabel manually, which doesn't seem great, given that it's a prop that they've passed in with the implicit understanding that we'd be handling application for them.